R/aging-data.R

```diff
@@ -9,35 +9,35 @@ AgingData <- R6::R6Class("AgingData",
   active = list(
     #' @field individualIds Array of individual ids (corresponding to the individualId column of the aging table)
     individualIds = function(value) {
-      private$wrapIntegerProperty("IndividualIds", value)
+      private$.wrapProperty("IndividualIds", value, asInteger = TRUE)
```

Comment:
`wrapPropert y` is implemented directly in rSharp:

https://github.com/Open-Systems-Pharmacology/rSharp/blob/2204ca57c5e8e84c733f3301f4ee9407a76c768f/R/net-object.R#L44